src/markdown-pages/terraform/terragrunt-configuration.mdx

```diff
@@ -0,0 +1,578 @@
+provider "newrelic" {
+ 	account_id = 12345 # Your New Relic account ID
+ 	api_key = "NRAK-***" # Your New Relic user key
+ 	region = "US" # US or EU (defaults to US)
+}
+```
+
+Tip: Documentation about the New Relic provider is provided on the [Terraform doc site.](https://registry.terraform.io/providers/newrelic/newrelic/latest/docs)
+
+Update the __provider.tf__ file to include your New Relic account information using variables keeping the Terraform configuration dynamic.
+```hcl
+variable "newrelic_personal_apikey" {}
+variable "newrelic_account_id" {}
+variable "newrelic_region" {}
+
+provider "newrelic" {
+ 	account_id = var.newrelic_account_id
+ 	api_key = var.newrelic_personal_apikey
+ 	region = var.newrelic_region
+}
+```
+
+In your terminal, change directories to the environment/dev folder and run the command `terragrunt init`:
+```bash
+terragrunt init
+```
+
+Terraform init sets up a bit of context, sets up environment variables, and then runs `terraform init`.
+
+```bash copyable=false
+[output] {muted}# Example output
+[output] ------------------------------------------------------------------------
+[output]
+[output] Terraform has created a lock file .terraform.lock.hcl to record the provider
+[output] selections it made above. Include this file in your version control repository
+[output] so that Terraform can guarantee to make the same selections by default when
+[output] you run "terraform init" in the future.
+[output]
+[output] Terraform has been successfully initialized!
+[output]
+[output] You may now begin working with Terraform. Try running "terraform plan" to see
+[output] any changes that are required for your infrastructure. All Terraform commands
+[output] should now work.
+[output]
+[output] If you ever set or change modules or backend configuration for Terraform,
+[output] rerun this command to reinitialize your working directory. If you forget, other
+[output] commands will detect it and remind you to do so if necessary.
+[output] [terragrunt] [/Users/jsius/Sites/deco/terragrunt-config/environments/dev] 2021/02/02 13:30:31 Copying lock file [output] from /Users/jsius/Sites/deco/terragrunt-config/environments/dev/.terragrunt-cache/e-PoBgWhdv3v8QGOtDQxS_WeYu4/
+[output] 69zjIFUfApJiUt8gFmi-6-dcPe8/.terraform.lock.hcl to /Users/jsius/Sites/deco/terragrunt-config/environments/dev
+```
+
+Run `terragrunt plan`:
+```bash
+terragrunt plan
+```
+
+Terragrunt starts to run but pauses to ask for the inputs to the variables added to the __provider.tf__ file.
+```bash copyable=false
+[output] {muted}# Example output
+[output] ------------------------------------------------------------------------
+[output]
+[output] 2/02 13:32:08 Running command: terraform plan
+[output] var.newrelic_account_id
+[output]  Enter a value:
+[output] Interrupt received.
+```
+
+Exit the `terragrunt plan` process. In the environments/dev __terragrunt.hcl__ file, add an inputs block to provide the values for your New Relic account vairables:
```

Comment:
```suggestion
Exit the `terragrunt plan` process. In the `environments/dev/terragrunt.hcl` file, add an `inputs` block to provide the values for your New Relic account variables:
```

Maybe its just me, but I'd prefer we output the full path so this is easy to follow. Also spotted a typo.

Reply:
CTRL + C to exit
```suggestion
Exit the `terragrunt plan` process with CTRL+C . In the `environments/dev/terragrunt.hcl` file, add an `inputs` block to provide the values for your New Relic account variables:
```